views/eApostilles/openEApp.ejs

```diff
@@ -0,0 +1,80 @@
+<% pageTitle="e-Apostille application" %>
+
+<div class="inner_header <%if (!user_data.loggedIn){%>no-user-signed-in<%}%>">
+    <a href="/dashboard" class="back-to-previous govuk-link">Back</a>
+    <%- partial ('../partials/inner-header.ejs')%>
+</div>
+<section class="column-three-quarters">
+    <span class="caption-xlarge">e-Apostille application</span>
+    <h1 class="heading-xlarge govuk-!-margin-top-2">
+        <%= applicationId %>
+    </h1>
+</section>
+<section class="column-two-thirds">
+    <p>
+        Your application has been successfully submitted.
+    </p>
+    <p>
+        We will notify you by email when it has been legalised. Your successfully legalised document will then
+        be
+        available for
+        download from this page.
+    </p>
+    <p>
+        You will have 21 days from when the documents have been successfully legalised to download them before
+        they are
+        deleted
+        for security reasons.
+    </p>
+
+    <table class="govuk-table right-align-last-cell ">
+        <caption class="heading-medium">PDF files</caption>
+        <tbody class="govuk-table__body">
+            <% for (const document of documents) {%>
+            <tr class="govuk-table__row">
+                <td class="govuk-table__cell"><%= document.name %></td>
+                <td class="govuk-table__cell">
+                    <strong class="govuk-tag govuk-tag--blue">
+                        <%= document.status %>
+                    </strong>
```

Comment:
one for the design team, maybe BA/product knowledge, but what are all of the statuses? is it OK that they are all blue, even though they might be different?

Reply:
So I originally thought there were only meant to be two types of blues but looks like red needs to be used as well. But those changes are to be done in a separate ticket https://jira.digital.csd.fcdo.gov.uk/browse/EIA-145 so this will all change at some point :)